mobile/src/components/home/DiscoverBar.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {View, Text, StyleSheet, Image, TouchableOpacity, Linking} from 'react-native';
+import {colors, globalStyleSheet} from '../../utilites/Theme';
+
+const DiscoverBar = () => {
+  return (
+    <View style={styles.DiscoverRow}>
+
+      <TouchableOpacity style={styles.container} onPress={() => Linking.openURL('https://isa.ualberta.ca/')}>
+        <Image
+          source={require('../../../assets/Website.png')}
+          style={styles.image}
+        />
+        <View style={styles.row}>
+          <Text style={[globalStyleSheet.text, styles.label]}>
+          ISA's Website
+          </Text>
+
+          <Image
+            source={require('../../../assets/Arrow.png')}
+            style={styles.image}
+          />
+        </View>
+      </TouchableOpacity>
+
+      <TouchableOpacity style={styles.container} onPress={() => Linking.openURL('https://isa.ualberta.ca/events')}>
+        <Image
+          source={require('../../../assets/Events.png')}
+          style={styles.image}
+        />
+
+        <View style={styles.row}>
+          <Text style={[globalStyleSheet.text, styles.label]}>
+          Events
+          </Text>
+
+          <Image
+            source={require('../../../assets/Arrow.png')}
+            style={styles.image}
+          />
+        </View>
+      </TouchableOpacity>
+
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    paddingVertical: 10,
+    paddingHorizontal: 16,
+    borderWidth: 1,
+    borderColor: colors.primary,
```

Comment:
if you look at the figma, it doesn't have a green border. Instead it has a light gray (#f7f8f7) background, no border color and a drop shadow effect around the container. We can use the current light gray in our theme file and we can change it later